### PR TITLE
[TASK] Add documentation for icons inside select field

### LINKED ISSUE #282: [DOCS] Documentation on how to add icons to the Select Type
Example:
```
  - identifier: icon
    type: Select
    renderType: selectSingle
    fieldWizard:
      selectIcons:
          disabled: false
    items:
      - value: facebook
        icon: social-media-facebook
      - value: instagram
        icon: social-media-instagram
      - value: linkedin
        icon: social-media-linkedin
      - value: youtube
        icon: social-media-youtube
```

### PR DESCRIPTION
**Pull request by PKuhlmay**

### CODE REVIEW

File: Documentation/YamlReference/FieldTypes/Select/Index.rst [hunk 1]
```diff
@@ -232,3 +255,30 @@ Select tree:
...
+
+
+..  _select-icons:
+
+Select with icons:
+
+.. code-block:: yaml
+
+   name: example/select
+    fields:
```
Comment (nhovratov) [inline Documentation/YamlReference/FieldTypes/Select/Index.rst]: The indentation looks off by one space

File: Documentation/YamlReference/FieldTypes/Select/Index.rst [hunk 2]
```diff
@@ -57,6 +57,29 @@ Settings
             - value: two
             - value: three
 
+   .. hint::
```
Comment (nhovratov) [inline Documentation/YamlReference/FieldTypes/Select/Index.rst]: maybe "tip" is better suited?

Comment (PKuhlmay) [reply Documentation/YamlReference/FieldTypes/Select/Index.rst]: I changed it to tip.

File: Documentation/YamlReference/FieldTypes/Select/Index.rst (hunk 1 above)
Comment (PKuhlmay) [reply Documentation/YamlReference/FieldTypes/Select/Index.rst]: I fixed it.

**Pull request merged**